src/main/java/seedu/address/model/appointment/AppointmentTime.java

```diff
@@ -23,7 +23,7 @@ public class AppointmentTime implements Comparable<AppointmentTime> {
      */
     public AppointmentTime(String time) {
         String dateString = time.substring(0, 10); // first 10 chars is the date
-        String rawTimings = time.substring(11);
+        String rawTimings = time.substring(10);
```

Comment:
The hours substring actually starts from position 10 instead of 11 due to a lack of space. To visualise this, simply put a breakpoint in this function.

Reply:
Refer to my comments above, then you can revert this change.